test/DataSet.test.js

```diff
@@ -1,44 +1,43 @@
 var assert = require('assert');
 var vis = require('../dist/vis');
-var moment = vis.moment;
 var DataSet = vis.DataSet;
 var Queue = vis.Queue;
 // TODO: test the source code immediately, but this is ES6
 
 var now = new Date();
 
 describe('DataSet', function () {
-  it('should work', function () {
-    // TODO: improve DataSet tests, split up in one test per function
+  var data = new DataSet({
+    type: {
+      start: 'Date',
+      end: 'Date'
+    }
+  });
 
-    var data = new DataSet({
-      type: {
-        start: 'Date',
-        end: 'Date'
-      }
-    });
+  // add single items with different date types
```

Comment:
See comment gelow on initialization of used data. In any case, don't leave this init code to run 'bare'.